Process_of_Authoring_GitHub_Actions_for_Azure.md

```diff
@@ -0,0 +1,97 @@
+# GitHub Actions
+
+[GitHub Actions](https://help.github.com/en/articles/about-github-actions) is a community-led and community-powered approach to enable developers to automate their code to cloud workflows directly from their repositories.
+There are now 6900+ community-developed actions available in [GitHub Marketplace](https://github.com/marketplace). 
+Since releasing Actions in November 2017, GitHub partners including Cloud partners like AWS, Azure, Google, and many others have also contributed to this list to help developers (teams) to extend and automate workflows with their existing tools.
+
+# GitHub Actions for Azure
+[GitHub Actions for Azure](https://github.com/Azure/actions) are released to GitHub Marketplace for creating workflows to package, release, and deploy apps to [Azure](https://azure.microsoft.com/) from GitHub. These actions enable developers to target the breadth of Azure services, from Web Apps to serverless Functions, as well as Azure SQL and MySQL databases. We also released Actions for building and deploying container-based applications that work with Docker and Kubernetes on any environment and cloud, in addition to their managed Azure Kubernetes Service.
+
+To help GH developers easily create GitHub CI/CD workflows targeting Azure, we also published [Azure starter templates](https://github.com/Azure/actions-workflow-samples) to deploy apps created with popular languages and frameworks such as .NET, Node.js, Java, PHP, Ruby or Python, in containers or running on any operating system.
+
+GitHub Actions for Azure are also integrated into Visual Studio Code, Azure CLI, and the Azure Portal simplifying the experience of [deploying to Azure from your favorite tools.](https://azure.microsoft.com/en-us/blog/deploy-to-azure-using-github-actions-from-your-favorite-tools/)
+
+This document is intended to help developers with the recommended process to create, publish and contribute to actions for Azure. Please reach out to the [vTeam](mailto:ace-team@github.com) for any queries.
```

Comment:
In all the places where the vTeam name is mentioned, should we instead use "ACE team" matching to the alias, so that in the markdown it will be clear which team it is?

Reply:
Can do that. But I assume we are all on same page that "ACE" as a brand needs to be introduced to all contributors as the team thats behind managing all of the gatekeeper processes for Azure Actions. Can we plz have that ACE manifesto included somewhere to act as a quick intro to ACE? it could be part of this doc or a separate one too. For now I will replace V team with ACE but will not include any details around scope of work ACE team is responsible for.